pkg/routeagent_driver/handlers/mtu/mtuhandler.go

```diff
@@ -170,3 +170,40 @@ func (h *mtuHandler) newNamedIPSet(key string, ipSetIface ipset.Interface) ipset
 		SetType: ipset.HashNet,
 	}, ipSetIface)
 }
+
+func (h *mtuHandler) Stop(uninstall bool) error {
```

Comment:
Overall, code looks OK.

I have one Q, what's the flow from uninstall to 'mtuhandler.Stop', I assume that the controller is responsible for that

Reply:
OK, found it, it's the uninstall flow 
https://github.com/submariner-io/submariner/blob/devel/pkg/event/registry.go#L86